packages/frameworks/frameworks.json

```
@@ -111,6 +111,34 @@
       }
     }
   },
+  {
+    "name": "Docusaurus 2",
+    "slug": "docusaurus-2",
+    "demo": "https://docusaurus.now-examples.now.sh",
+    "logo": "https://raw.githubusercontent.com/zeit/now/master/packages/frameworks/logos/docusaurus.svg",
+    "tagline": "Docusaurus makes it easy to maintain Open Source documentation websites.",
+    "description": "A static Docusaurus site that makes it easy to maintain OSS documentation.",
+    "website": "https://v2.docusaurus.io",
+    "detectors": {
+      "some": [
+        {
+          "path": "package.json",
+          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"@docusaurus\\/core\":\\s*\".+?\"[^}]*}"
+        }
+      ]
+    },
+    "settings": {
+      "buildCommand": {
+        "placeholder": "`npm run build` or `docusaurus build`"
+      },
+      "devCommand": {
+        "value": "docusaurus start --port $PORT"
+      },
+      "outputDirectory": {
+        "value": "build"
+      }
+    }
+  },
   {
     "name": "Docusaurus",
```

```suggestion
    "name": "Docusaurus 1",
```

Is this the name that shows up in the dropdown?

---

yes